### PR TITLE
Feature/article details

### LINKED ISSUE #124: Crear página Detalles del artículo
Crear la página de detalel del articulo siguiendo la línea de diseño del template actual.

Actualmente en Dev ya existe una línea que te lleva al artículo:

El link de los artículos debe dirigir a `/articulo?aid=[article_id]`
Ya existe la ruta `articulo` y el componente principal `page` para artículo

Hay que implementar esa `page` y crear el layout y componentes internos, así como la lógica.

También ya existe la función de la API que te devuelve un artículo por su ID:  `getArticule`

### PR DESCRIPTION
**Pull request by fdsosa**
Resolve #124

### CODE REVIEW

File: core/api.php [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 if ($request->action == ACTION_ADD_TO_CART) {
   setRequestData('aid', $request->aid);
+  setRequestData('qty', $request->qty);
   $status       = addToCart($request->qty);
```
Comment (Miguelslo27) [inline core/api.php]: Keep spaces at a minimum

File: core/api/cart.php [hunk 2]
```diff
@@ -7,6 +7,10 @@ function getCurrentCart()
 
 function addToCart($qty = 1)
 {
+  if((getRequestData('qty')) > 0){
```
Comment (Miguelslo27) [inline core/api/cart.php]: Is there a reason to wrap the `getRequestData(...)` inside parentheses?

File: core/api/cart.php [hunk 3]
```diff
@@ -7,6 +7,10 @@ function getCurrentCart()
...
+    $qty    = getRequestData('qty');
```
Comment (Miguelslo27) [inline core/api/cart.php]: Minimum spaces

File: core/api/cart.php [hunk 4]
```diff
@@ -50,6 +55,7 @@ function addToCart_checkIncommingData($qty)
     $status->errors[]  = 'Quantity should be a number';
     return $status;
   }
+  */
```
Comment (Miguelslo27) [inline core/api/cart.php]: Is this still giving an error or we can remove the comments at all? Just asking :)

File: template_dev/css/layout.css [hunk 5]
```diff
@@ -25,6 +25,7 @@
 html {
   font-size: 16px;
   overflow-x: hidden;
+  scroll-behavior: smooth;
```
Comment (Miguelslo27) [inline template_dev/css/layout.css:28]: Nice 👌

**Review by Miguelslo27 — CHANGES_REQUESTED** [review]
Some change requests, some doubts to ask, nothing big

File: core/api/cart.php (hunk 4 above)
Comment (fdsosa) [reply core/api/cart.php]: The variable `$qty` is a **string**, function `addToCart_checkIncommingData` test if `$qty` is a **integer** or **double**. We hace 3 options, change the type of `$qty`, modify the function or delete the test of  `$qty`

**Review by Miguelslo27 — APPROVED** [review]
Excelent work! 💪 Keep doing this

File: core/api/cart.php (hunk 4 above)
Comment (Miguelslo27) [reply core/api/cart.php]: Let's transform the qty to `integer` before the check, see `intval` native PHP function

**Pull request merged**